### ISSUE TITLE
SSL Issue : module 'ssl' has no attribute 'PROTOCOL_TLS_SERVER'

### ISSUE THREAD

**Issue by annibac** [Bearle/django-private-chat#25]
* django-private-chat version: 1.9
* Django version: 2.1
* Python version: 3.5
* Operating System: Linux

### Description

Hello !

I've been working with django private chat on a dev environement, and a prod without using ssl.
When I enabled SSL on my website, the chat didn't work anymore.
Here are my config in my settings:

```
CHAT_WS_SERVER_HOST = 'mysite.fr'
CHAT_WS_SERVER_PORT = 5002
CHAT_WS_SERVER_PROTOCOL = 'wss'
DJANGO_NOTIFICATIONS_CONFIG = { 'USE_JSONFIELD': True}
NOTIFICATIONS_USE_JSONFIELD = True
```

### What I Did

I change my systemd file and add the path to my .cer file but it didn't work, yet i get this error:

```
File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/home/annibal/.local/lib/python3.5/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/annibal/.local/lib/python3.5/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/annibal/.local/lib/python3.5/site-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/annibal/.local/lib/python3.5/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/home/annibal/.local/lib/python3.5/site-packages/django_private_chat/management/commands/run_chat_server.py", line 18, in handle
    ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
AttributeError: module 'ssl' has no attribute 'PROTOCOL_TLS_SERVER'
```
Could you help me please ?
Thank you,
Céline

**Comment by delneg** [comment]
Hello,
Looks like ssl.PROTOCOL_TLS_SERVER exists only in python 3.6.
I ran into this code in my pycharm:
`if sys.version_info < (3,) or sys.version_info >= (3, 4):
    PROTOCOL_TLSv1_1: int
    PROTOCOL_TLSv1_2: int
if sys.version_info >= (3, 5):
    PROTOCOL_TLS: int
if sys.version_info >= (3, 6):
    PROTOCOL_TLS_CLIENT: int
    PROTOCOL_TLS_SERVER: int`

It wasn't me who wrote the original WSS code, so I'll have to investigate further.
Is it possible for you to test this in a 3.6 environment?

**Comment by delneg** [comment]
https://docs.python.org/3.6/library/ssl.html#ssl.PROTOCOL_TLS
According to the docs, we should probably change that to use ssl.PROTOCOL_TLS for 3.5

**Comment by annibac** [comment]
Thank you, you solved my problem!
Céline.

**Comment by delneg** [comment]
No problem.
I'll add the code for 3.5 support later

**Comment by delneg** [comment]
Probably fixed in 0.2.0, please test
